frontend/src/metabase/account/profile/containers/UserProfileApp/tests/common.unit.spec.tsx

```diff
@@ -0,0 +1,14 @@
+import { screen } from "__support__/ui";
+
+import { setup } from "./setup";
+
+describe("UserProfileApp (OSS)", () => {
+  it("should show all form fields, even when password login is disabled", () => {
```

Comment:
[`UserProfileApp`](https://github.com/metabase/metabase/blob/e1fd8b38ea967f0fcc284a6d43ca8ad014bb3970/frontend/src/metabase/account/profile/containers/UserProfileApp/UserProfileApp.tsx#L16) depends on the [`getIsSsoUser`](https://github.com/metabase/metabase/blob/e1fd8b38ea967f0fcc284a6d43ca8ad014bb3970/frontend/src/metabase/account/profile/selectors.ts#L8) selector, which will return `true` if any of the predicate functions in `PLUGIN_IS_PASSWORD_USER` return `false`.

[The predicate function pushed above in `metabase-enterprise/auth/index.js`](https://github.com/metabase/metabase/pull/32644/files#diff-3b53e4f8f3766eb0c8035f4310b70fbc6c9aabaf55f8d04676d30978319cd74eR249) would normally return `false` on any instance where the password login setting was disabled. Now, however, it will only run and return `false` when the user has the `disable_password_login` token feature.